code/extensions/FileSubsites.php

```diff
@@ -9,6 +9,13 @@ class FileSubsites extends DataExtension {
 	// If this is set to true, all folders created will be default be
 	// considered 'global', unless set otherwise
 	static $default_root_folders_global = false;
+
+	/**
+	 * Show global folders in AssetAdmin
+	 * @config
+	 * @var bool
+	 */
+	private static $show_global_folders = true;
```

Comment:
This config can't be turned off via YML, due to merge rules ignoring falsey values being merged with truthy.

Could we flip the meaning so that the default is false? E.g. 'hide_global_folders = false'.